docs/md/README.md

```diff
@@ -13,68 +13,166 @@ is a workload manager.
+via the `deps.edn`, optionally specifying the kind:
+
+```shell
+clojure -A:test [unit|integration]
+```
+
+Note that the integration tests currently require a little more configuration
+before they can be run, namely, they require a `wfl` server running locally:
+
+```shell
+./ops/server.sh
 ```
-to only build the WFL and its docker images without running tests.
 
-See the output of `make help` for more information.
+### Deploy
+
+Currently, we mainly deploy WFL to gotc-dev and gotc-prod projects.
+When it's time to deploy WFL, developers need to bump the version string
+in the `version` file at the root of repo, which could be done either in
+a standalone PR or along with a feature PR. After having done that, the Github Action
+[Release Latest Version](https://github.com/broadinstitute/wfl/actions?query=workflow%3A%22Release+Latest+Version%22)
+will get triggered to build and push the tagged docker images of WFL to
+[DockerHub](https://hub.docker.com/repository/docker/broadinstitute/workflow-launcher-api).
 
-## Versioning
+From here, the developers who have broad VPN connected can go to the
+[Jenkins Page](https://gotc-jenkins.dsp-techops.broadinstitute.org/job/deploy-wfl/build?delay=0sec)
+to deploy applicable versions of WFL to various available cloud projects.
 
-Workflow Launcher needs to manage its version and the versions of
-`dsde-pipelines.git` which contribute the WDL files. There may be as
-many `dsde-pipelines.git` versions as there are workflow wdls.
+!!! warning
+    In addition to its own version, Workflow Launcher also needs to manage
+    the verions of `dsde-pipelines.git` which contribute the WDL files.
+    Currently, that version is controlled by the commit hash string in
+    function `stage-some-files` in `api/src/boot.clj`, for instance:
 
-The `wfl` jar includes a manifest with at least that information in
-it, and a version command that returns it.
+    ```clojure
+    (util/shell-io! "git" "-C" (.getParent environments)
+                      "checkout" "ad2a1b6b0f16d0e732dd08abcb79eccf4913c8d8")
+    ```
 
-## Capabilities
+    In the long term, this is likely to change.
 
-When Workflow Launcher is on-premises or in the cloud, it can
-currently talk to the following services:
+Learn more about the deployment details in
+[Deployment of WorkFlow Launcher](/docs/md/dev-deployment.md).
 
-| service                     | on premises | in cloud |
-| --------------------------- | ----------- | -------- |
-| Cloud SQL                   | x           | x        |
-| Cromwell                    | x           | x        |
-| Google App Engine           | x           | x        |
-| Google Cloud Platform Admin | x           | x        |
-| Google Cloud Storage        | x           | x        |
-| Oracle DB                   | x           |          |
-| Vault                       | x           | x        |
+### Diagnosis
 
-Workflow Launcher has a diagnostic mode, `dx`,
+Workflow Launcher has a diagnostic command, `dx`,
```

Comment:
I wonder if we should make this undocumented as this really only makes sense in command like mode (aka zero). I'm unsure if we want to keep this for a server?

Reply:
This is a really good point! I hesitated about this a bit, but now I think it makes sense to remove it from here. (At least if we wanna keep it, it should be in the dev section not readme.)